packages/js/src/dashboard/transformers/difference.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {number} number The number to check.
+ * @returns {boolean} True if not falsy except 0.
+ */
+const isValidNumber = ( number ) => ! number && number !== 0;
+
+/**
+ * @param {number} current The current.
+ * @param {number} previous The previous.
+ * @returns {number} The difference, as percentage.
+ */
+export const getDifference = ( current, previous ) => {
+	// Invalid values (falsy except 0) => return NaN.
+	// Because there is nothing to say about the difference.
+	if ( isValidNumber( current ) || isValidNumber( previous ) ) {
```

Comment:
That means that if one of values is not a number we are fine with that?

Reply:
Oops, that should be `isInvalidNumber`